### ISSUE TITLE
Deploy gradle plugin to plugins.gradle.org

### ISSUE THREAD

**Issue by demtnt** [OpenAPITools/openapi-generator#2265]
### Is your feature request related to a problem? Please describe.

It would be nice to start publishin gradle plugin to plugins.gradle.org to be able to use plugin block in build.gradle. e.g.
`
plugins {
    id 'org.openapi.generator' version '1.2.3'
}
`

## Describe the solution you'd like

Deploy to plugins.gradle.org

## Describe alternatives you've considered

Leave as it is.

**Comment by wing328** [comment]
Definitely a good idea. I'll see if I can do it next weekend.

cc @jimschubert

**Comment by jimschubert** [comment]
@wing328 I had started looking at this not too long ago. There was an article about an XSRF issue on the site posted on the day I began looking, and it made me stop to research it further. Things came up, and I didn't research it, but it seems it was fixed by the time of the article.

see: https://medium.com/bugbountywriteup/gradle-plugin-portal-clickjacking-cross-site-request-forgery-enabling-account-takeover-d65c2e43082b
